core/src/toga/widgets/base.py

```diff
@@ -17,10 +19,29 @@
 StyleT = TypeVar("StyleT", bound=BaseStyle)
 
 
+# based on colors from https://davidmathlogic.com/colorblind
+debug_background_palette = [
```

Comment:
If this is a list of constants, it should be treated as a constant by naming convention:
```suggestion
DEBUG_BACKGROUND_PALETTE = [
```